services/skus/controllers.go

```diff
@@ -588,6 +589,47 @@ func CreateOrderCreds(svc *Service) handlers.AppHandler {
 	}
 }
 
+// deleteItemCreds handles requests for deleting credentials for an item.
+func deleteItemCreds(svc *Service) handlers.AppHandler {
+	return func(w http.ResponseWriter, r *http.Request) *handlers.AppError {
+		ctx := r.Context()
+		lg := logging.Logger(ctx, "skus.deleteItemCreds")
+
+		orderID := &inputs.ID{}
+		if err := inputs.DecodeAndValidateString(ctx, orderID, chi.URLParamFromCtx(ctx, "orderID")); err != nil {
+			lg.Error().Err(err).Msg("failed to validate order id")
+			return handlers.ValidationError("Error validating request url parameter", map[string]interface{}{
+				"orderID": err.Error(),
+			})
+		}
+
+		itemID := &inputs.ID{}
+		if err := inputs.DecodeAndValidateString(ctx, itemID, chi.URLParamFromCtx(ctx, "itemID")); err != nil {
+			lg.Error().Err(err).Msg("failed to validate item id")
+			return handlers.ValidationError("Error validating request url parameter", map[string]interface{}{
```

Comment:
suggestion: improve the readability of the error message.

If we look the [`handlers.ValidationError()`](https://github.com/brave-intl/bat-go/blob/9f0b7aa487086dd4b0769f437a2ae5114a29a520/libs/handlers/handlers.go#L116) function it concats the passed in message with `Error validating` so we end up with a message like `Error validating Error validating request url parameter` which does not read very well. 

Can this be changed?





`Error validating Error validating request url parameter`

Reply:
addressed: https://github.com/brave-intl/bat-go/pull/2281/commits/a40fb6cdf80029cc56e66e28a0ede7d70f191587